maas-schemas/schemas/core/components/costBreakDown.json

```diff
@@ -0,0 +1,14 @@
+{
+  "$id": "https://schemas.maas.global/core/components/costBreakDown.json",
+  "description": "A costBreakDown shows pricing according to TSPs wishes and nuances",
+  "definitions": {
+    "costBreakDownArray": {
+      "type": "array",
+      "description": "TSP specific cost rules",
+      "items": {
+        "type": "string"
+      },
+      "examples": [["First 10 km, 50 euros", "Next 20 km, 10 euros"]]
```

Comment:
Accepted by Frontend guys (Manider & Duc)

Reply:
isn't this fare ? As cost is something we pay to TSP. And fare is something that user needs to pay maas ?

Reply:
TSP always returns `cost` - and `costBreakDownArray` is a description telling how this cost is counted.

Reply:
So how this is actually used is it just proxied to client and displayed on UI ? 
If yes are fare rules applied to it on backend ? I.e if we apply % discount to Taxi will be `costBreakDownArray` also discounted ?

Reply:
Yes - this property says only about `costs` taken by the TSP provider - not how much user pays. Its mandatory by some TSPs. For example for `nearme` we need to display what was the price for the trip : 
a) 100 euro (on that commision fee 20 euro)
b) 100 euro (prommotional discount 10 euro)

And we need to display that if user paid for the trip or not (because he had some subscription).

Reply:
Correct me if I'm wrong, but this is just a description, i.e. it's for information purposes or is this build dynamically?

Anyway... if that's just details that are going to be shown in UI then maybe you just need to rename the property as @vonpoland suggested to `fareBreakdownDetails` or `fareBreakdownInformation`.

The more important thing is that the string should be also translated in to given locale or passed to UI as translation keys but overall, schema looks okay.

Reply:
@iaforek  - its dynamic + it will be already localized in TSP.

We should not change from cost to fare, because it is not `fare`. 

Example: 
  - user pays 10 euro because has some plan (fare = 10)
  - tsp cost is 20 (cost = 20)
  - for some tsp we need to display what was the real cost of the booking

Reply:
Actually, after second thought, this property could be simply named: `fareRules` :)

Reply:
Don't use fareRules or any existing term please. It's defined term for something else